pages/common/nmap.md

```diff
@@ -4,6 +4,16 @@
 > Some features only activate when Nmap is run with privileges.
 > More information: <https://nmap.org>.
 
+- If Nmap throws a root privileges error use it with `sudo` or `admin rights cmd`
+
+- To check if the IP address `192.168.1.101` is up, with it’s operating system info. It may show RTTVAR logs (for TCP) for a long time:
+
+`nmap -O 192.168.1.101`
+
+- To scan the Network for devices (including routers) and get info on their operating system:
+
+`sudo nmap -sS -O 192.168.1.0/24`
```

Comment:
Missing token syntax
```suggestion
`sudo nmap -sS -O {{192.168.1.0/24}}`
```

Reply:
Check this out:
```suggestion
`sudo nmap -sS -O {{IP/port}}`
```

Reply:
Hrm. It's not actually a port - it's a CIDR block number. We could do this?
```suggestion
`sudo nmap -sS -O {{ip_address}}/{{CIDR_block}}`
```
.....but it's not obvious what you're supposed to put there.  To that end, I think a concrete example would be best for this example:
```suggestion
`sudo nmap -sS -O {{192.168.1.0}}/{{24}}`
```